dace/subsets.py

```diff
@@ -576,17 +576,29 @@ def compose(self, other):
         else:
             raise NotImplementedError
 
-    def squeeze(self):
+    def squeeze(self, ignore_indices: List[int] = []):
```

Comment:
Mutable default parameters are not good. Needs `= None`  and `ignore_indices = ignore_indices or []` within the function.